### PR TITLE
Personography merge feature

### PR DESCRIPTION
**Pull request by rlskoeser**
Custom admin action to merge duplicate people records in the personography.  Combines account and person data into the selected primary person record, and deletes the merged account and person records.  More details in #72.

Does not yet include tests for the merge view; but with the refactor to move the actual merge logic into a queryset method (which is now fully tested), that should be relatively straightforward.

### CODE REVIEW

File: mep/people/views.py [hunk 1]
```diff
@@ -149,3 +155,54 @@ def get_queryset(self):
...
+    def get_form_kwargs(self):
+        form_kwargs = super(PersonMerge, self).get_form_kwargs()
+        form_kwargs['person_ids'] = self.person_ids
+        return form_kwargs
+
+    def get_initial(self):
+        # default to first person selected (?)
+        # _could_ add logic to select most complete record,
+        # but probably better for team members to choose
+        self.person_ids = [int(pid) for pid in self.request.GET.getlist('ids')]
```
Comment (bwhicks) [inline mep/people/views.py]: I believe this is not working as it stands (having tested it and gotten a ValueError). `.getlist()` assumes that the querystring would be like `?ids=366&ids=365`, whereas this gets a single element list with a string that's `366,365` for example.

**Comment by rlskoeser** [comment]
@bwhicks thanks for the helpful feedback.  I've pushed small changes to address the two issues you've found.  Still need to write unit tests for the view and add logic to document merged MEP ids.

**Comment by bwhicks** [comment]
Once the final unit tests are in, this all looks good, I think.

**Comment by rlskoeser** [comment]
@bwhicks I've updated the pull request with tests for the view, restricted the view to users with appropriate permissions, added logic to preserve merged MEP ids in the notes, and handled one more possible error case that I'd noted as `TODO` in a comment (person with a shared library account).

**Pull request merged**